spec/models/ingredient_spec.rb

```diff
@@ -12,4 +15,11 @@
       it { should have_many(:recipes).through(:recipe_ingredients) }
    end
 
+   describe "class methods" do
+      describe "#alpha_ingredients" do
+        it "can list all ingredients in alphabetical order" do
+          expect(Ingredient.alpha_ingredients).to eq([@tomato, @bread, @bun, @beef, @cheese])
```

Comment:
Before I saw your test data, I was a bit concerned about why tomato would be before bread alphabetically 😂